WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanViewModel.kt

```diff
@@ -97,12 +99,12 @@ class ScanViewModel @Inject constructor(
         }
     }
 
-    private fun fetchScanState(startWithDelay: Boolean = false, isRetry: Boolean = false) {
+    private fun fetchScanState(invokedByUser: Boolean = false, isRetry: Boolean = false) {
```

Comment:
Suggestion (💡): This is a suggestion for the future, but obviously just something you already doing well. I would have committed this `invokedByUser` change separately, as a `Refactor` commit. I would do that just to separate this commit from the `Feature` related commit just to make it easier for the reviewer to understand immediately that this has nothing to do with the actual feature itself.

Reply:
I agree, a separate commit must've been better here. I'll keep this in mind for the future. Thank you for this suggestion 🙏.